bin/ch-grow

```diff
@@ -584,6 +596,20 @@ def cp(src, dst):
    else:
       shutil.copy2(src, dst)
 
+def dependencies_check():
+   # Check some more dependencies. If any dependency problems found, here or
+   # above (e.g., lark module checked at import time), the complain and exit.
+   if (not os.path.exists(CH_RUN)):
+      depfails.append(("missing", CH_RUN))
+   if (not shutil.which("skopeo")):
+      depfails.append(("missing", "skopeo"))
+   if (not shutil.which("umoci")):
+      depfails.append(("missing", "umoci"))
+   for (p, v) in depfails:
+      ERROR("%s dependency: %s" % (p, v))
+   if (len(depfails) > 0):
+      sys.exit(1)
+
```

Comment:
Should we be enforcing some minimum version for these dependencies?

Reply:
Yes, but I don't know what they are for the moment.